deploy/kubernetes/helm-charts/altinn-designer/templates/tests/test-connection.yaml

```diff
@@ -14,5 +14,5 @@ spec:
     - name: wget
       image: busybox
       command: ['wget']
-      args:  ['altinn-designer:{{ .Values.service.internalPort }}']
+      args:  ['altinn-designer:{{ .Values.image.containerPort }}']
```

Comment:
This should use the service's port, not the containers port. Because we have the same ports on both service and deployments/pods this works for now. But something we should keep in mind moving forward.

Reply:
Changed to .Values.service.externalPort (in all these references).